src/palimpzest/prompts.py

```diff
@@ -1,6 +1,7 @@
 """This file contains prompts used by Palimpzest
-Whenever they are called, they can be parameterize with the str.format() method using the parameter names that are in brackets.
-For now, this is an easy decoupling. In the future, we maybe want a more sophisticated approach like a PromptBuilder.
+Whenever they are called, they can be parameterize with the str.format() method using the parameter names that are in
+brackets. For now, this is an easy decoupling. In the future, we maybe want a more sophisticated approach like a
+PromptBuilder.
```

Comment:
For this entire file, I would revert all changes that keep lines within 120 characters; the reason is that these prompts are inside of `"""`, which means that moving things to the next line actually adds a `\n` to the prompt, which may affect performance of models on these prompts.

(They may perform better, or worse, but I'd rather not create any issues like that now which would need to be checked for a regression)

Based on what you've told me earlier, it sounds like `Ruff` should allow us to specify that -- just for this file -- our line character limit is infinite? (or a really large number?)

Reply:
Reverting whole change to prompts